app/config/config.yml

```diff
@@ -9,6 +9,9 @@ imports:
     - { resource: security.yml }
     - { resource: services.yml }
 
+parameters:
+    locales: en|fr|de|es|cs|ru|uk|ro|pt_BR
```

Comment:
I wonder if `locales` is a too generic name. Should we rename it to `enabled_locales`, `active_locales`, `app_locales`, etc.?